### DISCUSSION TITLE
Is it possible to name the hubitats when you have more than one?

### DISCUSSION THREAD

**Discussion by simondsmason** [jason0x43/hacs-hubitat#202] [Q&A]
When I add a second Hubitat hub I see it gives it an alphanumeric name but it is difficult to tell which hub is which.  Is it possible to use the hub name or to name the hub in the integration?  Thanks.

**Comment by jason0x43** [comment]
On the Integrations screen in HA, each integration card has a three-dot menu. The "Rename" option in there will let you rename an instance of an integration.

<img width="299" alt="Screenshot 2023-04-09 at 08 45 51" src="https://user-images.githubusercontent.com/1042866/230773286-bf0d144a-c009-474d-af2a-612eefa12dca.png">

**Comment by simondsmason** [comment]
Sorry, I don't think it could be any more obvious - not sure why I missed it.  Hopefully my idiot question will help someone else in the future.  Thanks.
